tests/functional/storageclass/test_cross_sc_clone_snap_restore.py

```diff
@@ -128,3 +118,118 @@ def test_cross_class_same_pool_clone_snap_restore(
         restore_pvc_obj2.reload()
         log.info("PVC was restored from the snapshot on another SC")
         restore_pvc_obj2.delete()
+
```

Comment:
Please skip the test case on provider mode and managed service. 
```
@skipif_managed_service
@skipif_hci_provider_and_client

```

Reply:
@jilju Are those skips relevant only for test cases of clone/restore to a sc in a DIFFERENT pool ? Or I should I also skip the test case in line 33 ( clone/restore to a different storage class but on the same pool) ?

Reply:
The current design of the test will not work in provider mode because we need to use storageclaim to create storageclass and cephblockpool.

Reply:
Done, thank you!